FirstProject/css/main.css

```diff
@@ -80,11 +84,66 @@ h2.description_section {
     margin-left: 300px;
 }
 
-/*.row_section {*/
-    /*margin-left:254px;*/
-    /*margin-right: 255px;*/
-/*}*/
-
 fieldset.field_section {
     text-align: center;
-}
\ No newline at end of file
+}
+
+legend {
+    text-align: left;
+    color: #03A9F4;
+}
+
+input[type='radio']{
+    width: 35px;
+    height: 23px;
+    border-radius: 15px;
+    top: -2px;
+    left: -1px;
+    position: relative;
+    background-color: #d1d3d1;
+    content: '';
+    display: inline-block;
+    visibility: visible;
```

Comment:
display: inline-block; and visibility: visible; are not required because the buttons will have this properties by default.